dev-environment/nginx/pmml/templates/default.conf.template

```diff
@@ -0,0 +1,30 @@
+upstream pmml {
+    server ${PMML_SAMPLES_URL};
+}
+
+server {
+    listen ${NGINX_PMML_STATIC_PORT};
+    server_name static-data;
+
+    location /static/int {
+        rewrite /static/int /int break;
+    }
+    location /static/double {
+        rewrite /static/double /double break;
+    }
+    location /static/boolean {
+        rewrite /static/boolean /boolean break;
+    }
+    location /static/string {
+        rewrite /static/string /string break;
+    }
+}
```

Comment:
this is not meant to be served for pmml - we use this data to test the model proxy which can add extra parameters to model so it is should be separated